docs/general/builtins/comparison-operators.rst

```diff
@@ -85,11 +85,11 @@ normalized to MultiLineString, MultiPoint and MultiPolygon. Hence, geometry
 collection of points is equal to a MultiPoint with the same points set::
 
     cr> select 'MULTIPOINT ((10 40), (40 30), (20 20))'::GEO_SHAPE = 'GEOMETRYCOLLECTION (POINT (10 40), POINT(40 30), POINT(20 20))'::GEO_SHAPE;
-    +------+
-    | true |
-    +------+
-    | TRUE |
-    +------+
+    +-----------------------------------------------------------------------------------------------------------------------------------------------------+
+    | (CAST('MULTIPOINT ((10 40), (40 30), (20 20))' AS geo_shape) = CAST('GEOMETRYCOLLECTION (POINT (10 40), POINT(40 30), POINT(20 20))' AS geo_shape)) |
+    +-----------------------------------------------------------------------------------------------------------------------------------------------------+
+    | TRUE                                                                                                                                                |
+    +-----------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Maybe better to add an alias to the column instead for readability?

Reply:
done!